test/extended/cpu_partitioning/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+  - eggfoobar
+reviewers:
+  - yanirq
```

Comment:
do we want more reviewers?

Reply:
please add @MarSik as well to the reviews list and others that can fit